client/components/gsuite/docs/new-user-list.tsx

```diff
@@ -9,24 +9,33 @@ import {
 	GSuiteNewUserField,
 	newUsers,
 } from 'calypso/lib/gsuite/new-users';
+import type { SiteDomain } from 'calypso/state/sites/domains/types';
 
-const domainOne = { name: 'example.blog' };
-const domainTwo = { name: 'test.blog' };
+const domainOne: SiteDomain = {
+	name: 'example.blog',
+	domain: 'example.blog',
+	supportsGdprConsentManagement: true,
+};
+const domainTwo: SiteDomain = {
+	name: 'test.blog',
+	domain: 'test.blog',
+	supportsGdprConsentManagement: true,
+};
 
 function GSuiteNewUserListExample(): JSX.Element {
-	const [ users, setUsers ] = useState( newUsers( domainOne.name ) );
+	const [ users, setUsers ] = useState( newUsers( domainOne.name ?? '' ) );
 	const [ domains, setDomains ] = useState( [ domainOne ] );
 	const [ useMultipleDomains, setUseMultipleDomains ] = useState( false );
 	const [ useExtraValidation, setUseExtraValidation ] = useState( false );
 
 	const toggleUseMultipleDomains = () => {
 		if ( useMultipleDomains ) {
 			setDomains( [ domainOne ] );
-			setUsers( newUsers( domainOne.name ) );
+			setUsers( newUsers( domainOne.name ?? '' ) );
```

Comment:
The Nullish coalescing operator ?? is redundant here. `domainOne` clearly has a defined name.

Reply:
Scratch this. I forgot to look at the definition for `newUsers`